### PR TITLE
Test and Coverity fixes

### PR DESCRIPTION
**Pull request by ishche**

### CODE REVIEW

**Review by grianbrcom — APPROVED** [review]
:wolf:

**Review by temanbrcom — CHANGES_REQUESTED** [review]
Missing semantic tags on the commit messages

**Comment by grianbrcom** [comment]
Could you rename your commits with "fix:"?
Looks like the Coverity offer to make the type field static.

**Comment by ishche** [comment]
> Looks like the Coverity offer to make the type field static.

Can't do that, this will prevent the type to be serialized to JSON.

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/RootNode.java [hunk 1]
```diff
@@ -14,16 +14,12 @@
...
-import lombok.ToString;
 import lombok.Value;
 import org.eclipse.lsp.cobol.core.model.Locality;
 
 /**
  * The class represents the root. All trees must start with one root node.
  */
-@ToString
-@EqualsAndHashCode
 @Value
```
Comment (temanbrcom) [inline server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/RootNode.java:23]: Shouldn't it also have the `@EqualsAndHashCode(callSuper = true)`?

**Comment by grianbrcom** [comment]
> Can't do that, this will prevent the type to be serialized to JSON.
Type can be a part of CFASTNode.

**Review by asatklichov — APPROVED** [review]
🏯

**Comment by temanbrcom** [comment]
This is not a user bug yet, please change the prefix either to "refactor" or to "style" for the first and the last commit

**Pull request closed without merging**